app/models/custom_attribute.rb

```diff
@@ -2,7 +2,17 @@ class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   serialize :serialized_value
 
+  def value=(value)
```

Comment:
@alongoldboim yes, I think that's essentially the same concern. Re: the public API, if I do `some_object.serialized_value = some_thing`, that looks like it's relying on me to do the serialization for it. I'd want to do `some_object.value = some_thing` and have AR handle the serialization and deserialization, so I'd never have to be concerned with serialization outside of the class. I think that's what @Fryguy is saying in that comment too, unless I'm mistaken